scripts/binary/binary-integrity-check-source.js

```diff
@@ -143,13 +179,18 @@ function integrityCheck (options) {
         fileName: 'evalmachine.<anonymous>',
       },
       {
-        functionName: 'Module2._extensions.<computed>',
+        functionName: 'c._extensions.<computed>',
+        methodName: '.jsc',
         // eslint-disable-next-line no-undef
         fileName: [appPath, 'index.js'].join(PATH_SEP),
+        line: 1,
+        column: 2076,
```

Comment:
what are the scenarios where this column number would change? new imports into the root file?

Reply:
This would change if our entry point file bundle changed at all. I don’t see that happening very frequently though and if it does, we will just update these numbers. The situations where we would update the entry point bundle is if this file changes: https://github.com/cypress-io/cypress/blob/develop/scripts/binary/binary-entry-point-source.js or if bytenode gets updated.

Reply:
will this error provide insights into that column number or do we just fail and we'd have to manually verify?

Reply:
What sort of insights would you expect? The error will let us know what that there was a difference and what the difference is (similar to what we were doing with file and function names previously).

Reply:
sorry - i didn't remember what we were prev doing. I was just curious if this error outputs on any download or just during the build process

Reply:
This error will output when the binary is corrupted. So if a user tries to open Cypress with a bad binary, they will see "your code executed from column x instead of y"